File: Primer-Bimestre/Primer Parcial/IP-Taller2/IP-Taller2_python/TipoDatos.py
print("")
print("***Programa para conocer los datos en python***")
print("")

# Declarar las variables de tipo entero
num1 = 10
num2 = 13

# Declarar variables de tipo decimal
num3 = 45.789
num4 = 23.9

# Declarar variables de tipo caracter
encender = 'S'
encender = 'N'

# Declarar variables de tipo cadena
cadena = "Ana Carolina"
apellido = "Churo Briceño"

#Operadores aritméticos

suma = 10 + 15
suma = num1 + num2
print(suma)

print("La suma es:" ,suma, "puntos")

suma1 = 14 + 34
print(suma1)

mult = 13 * 23
mult1 = num1 * num2
print(mult1)

div = 34.5/12.4
div1 = num1/num2
print(div1)

print("--Funciones Matemáticas--")
print(pow(2,4))
print(max(12,10))
print(min(45,1))

# operadores condicionales

print("-- Operadores condicionales --")
a = 34
b = 7

print(a > b)
print(a >= b)
print(a < b)
print(a <= b)
print(a == b)
print(a !=b)

# Operadores lógicos
print("-- Operadores Lógicos --")
print("--Conjunción -- AND --")
print(True & True)          #True
print(True & False)         #False
print(False & False)        #False

print("--------------------------")
print((a > b) & (a >= b))          #True
print((a == b) & (b > a))          #False

print("-- Disyunción -- OR --")
print(True|True)                   #True
print(True|False)                  #True
print(False|False)                 #False

print("-- Negación --")
print(not True)                   #False
print(not False)                  #True

File: Primer-Bimestre/Primer Parcial/IP-Taller2/IP-Taller2_java/src/ip/taller2/TipoDatos.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ip.taller2;

/**
 *
 * @author DELL
 */
public class TipoDatos {
    public static void main(String[] args) {
        
        //Deraclarando variables tipo entero
        int num1;
        num1 = 10;
        int num2;
        num2 = 13;
        
        //Declarar variables de tipo decimal
        double num3;
        num3 = 45.789;
        double num4 = 23.9;
        
        //Declarar variables de tipo char
        char encender ='S';
        encender = 'N';
        
        //Declarar variables de tipo cadena
        String cadena;
        cadena = "Ana Carolina";
        
        String apellido = "Churo Briceño";
        
        //Operadores aritméticos
        
        int suma;
        suma = 10 + 15;
        suma = num1 + num2;
        System.out.println(suma);
        
        System.out.println("La suma es:" +suma + " puntos");
    
        int suma1 = 14 + 34;
        
        System.out.println(suma1);
        
        double mult = 13 * 23;
        double mult1 = num1 * num2;
        
        System.out.println(mult1);
        
        double div = 34.5 / 12.4;
        double div1=num1/num2;
      
        System.out.println(div1);
        
        System.out.println("--Funciones Matemáticas--");
        System.out.println(Math.pow(2,4));
        System.out.println(Math.max(12,10));
        System.out.println(Math.min(45,1));
        
         //operadores condicionales
         System.out.println("-- Operadores condicionales --");
         int a = 34;
         int b = 7;
         
         System.out.println(a > b); //true
         System.out.println(a >= b); //true
         System.out.println(a < b); //false
         System.out.println(a <= b); //false
         System.out.println(a == b); //false
         System.out.println(a !=b); //true
         
         //operadores lógicos
         System.out.println("-- Operadores Lógicos --");
         System.out.println("--Conjunción -- AND --");
         System.out.println(true && true);//true
         System.out.println(true && false); //false
         System.out.println(false && false); //false
         
         System.out.println("--------------------------");
         System.out.println((a > b) && (a >= b)); // true
         System.out.println((a == b) && (b > a)); // false
         
         System.out.println("-- Disyunción -- OR --");
         System.out.println(true||true);//true
         System.out.println(true || false); //true
         System.out.println(false || false); // flase
         
         System.out.println("-- Negación --");
         System.out.println(!true); //false
         System.out.println(!false); //true           
    }
}
